### PR TITLE
Upgrade ember

### PR DESCRIPTION
**Pull request by alechirsch**
Solves #9 and #10

### CODE REVIEW

**Comment by efx** [comment]
Quick check looks good to me.

**Comment by efx** [comment]
Thank you for migrating the addon @alechirsch !

File: app/styles/addon.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .json-pretty pre {
-    outline: 1px solid #bbb;
```
Comment (efx) [inline app/styles/addon.css]: nitpick: whitespace changes seem unrelated; ideally we just copy the original file so it doesn't appear in the diff as a change

Comment (alechirsch) [reply app/styles/addon.css]: That is strange, I moved the file, didnt touch the contents

Comment (alechirsch) [reply app/styles/addon.css]: Ahhh, my environment is set up to use tabs vs. spaces

Comment (alechirsch) [reply app/styles/addon.css]: Would like like me to try to revert?

Comment (efx) [reply app/styles/addon.css]: If you wouldn't mind, yes please! As an aside, we could add the `.editorconfig` from the addon blueprints: https://github.com/ember-cli/ember-addon-output/blob/master/.editorconfig

Comment (alechirsch) [reply app/styles/addon.css]: Should be all set

**Comment by alechirsch** [comment]
@efx @bguiz Can this get merged?

**Comment by efx** [comment]
I'll merge and we should probably release a new version for users to consume. Assuming this is needed for #10
